src/components/collapsible-motion/collapsible-motion.js

```diff
@@ -2,7 +2,16 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import invariant from 'tiny-invariant';
 import { keyframes, ClassNames } from '@emotion/core';
-import Collapsible from '../collapsible';
+import isNil from 'lodash/isNil';
+import useToggleState from '../../hooks/use-toggle-state';
+
+const usePrevious = value => {
+  const ref = React.useRef();
+  React.useEffect(() => {
+    ref.current = value;
+  });
```

Comment:
You want this on every render or only when `value` changes?

Reply:
good call, should have [value] as deps array